### DISCUSSION TITLE
A way to see if the person you've been seeing from the app is still actively looking on the app.

### DISCUSSION THREAD

**Discussion by WilliamDormer** [PullDating/Org#67] [Ideas & Features]
Otherwise, how do you know if they have any intention of going steady, without explicitly asking them? Regardless it is a good gauge.

**Comment by ajbott** [comment]
I think a general "Relationship status" feature would work great for this. I think it's okay for people to see multiple people at once, but if they ever become exclusive, they can enter into a "relationship" and have their account linked to each other. This would prevent the other person from looking for additional matching if they are in a monogamous relationship. This would also inform the partner if they ever decide to exit the relationship. This would also inform other people on their match list that they are in a relationship and can no longer talk to you anymore.
